runtime/j9vm/j9vmnatives.xml

```diff
@@ -420,4 +420,9 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<export name="JVM_VirtualThreadHideFrames"/>
 	</exports>
 
+	<exports group="jdk21">
+		<!-- Additions for Java 21 (General) -->
+		<export name="JVM_VirtualThreadMount"/>
+		<export name="JVM_VirtualThreadUnmount"/>
+	</exports>
```

Comment:
This is not enough, `runtime/j9vm/module.xml` and `runtime/redirector/module.xml` must export this new group.

Reply:
Added, also discovered
* https://github.com/eclipse-openj9/openj9/pull/17012